packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -592,6 +648,10 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		return trees;
 	}
 
+	public setRevisionsForLoadedData(revision: RevisionTag): void {
```

Comment:
```suggestion
	public setTipRevisionForLoadedData(revision: RevisionTag): void {
```
Please add some doc comments to explain what this does and why/when it needs to be called.